src/Observable.ts

```diff
@@ -247,7 +247,7 @@ export class Observable<T> implements CoreOperators<T>  {
   publishBehavior: (value: any) => ConnectableObservable<T>;
   publishReplay: (bufferSize?: number, windowTime?: number, scheduler?: Scheduler) => ConnectableObservable<T>;
   publishLast: () => ConnectableObservable<T>;
-  race: (...observables: Array<Observable<any>>) => Observable<any>;
+  race: (...observables: Array<Observable<T>>) => Observable<T>;
```

Comment:
Just came to thought (I know it's not related with this PR) - if signature to be perfect parity compatible to RxJS4's `amb`, it would be something like
`race: (...observables: Array<Observable<T> | Promise<T>>) => Observable<T>;`
but `race` doesn't accept promise in its implementation yet.

Is my understanding correct?